csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/FileContent.cs

```diff
@@ -173,6 +195,14 @@ private void DoInitialize()
                             || line.Contains("<ImplicitUsings>enable</ImplicitUsings>".AsSpan(), StringComparison.Ordinal)
                             || line.Contains("<ImplicitUsings>true</ImplicitUsings>".AsSpan(), StringComparison.Ordinal);
 
+                        // Determine if WPF is used.
+                        useWpf = useWpf
+                            || line.Contains("<UseWPF>true</UseWPF>".AsSpan(), StringComparison.Ordinal);
```

Comment:
I wonder if we should use `OrdinalIgnoreCase` throughout instead?

Reply:
Seems to make sense. We already do case insensitive matching in regexes.